android/java/res/layout/rewards_panel_vbat_expire_notice.xml

```diff
@@ -78,14 +78,17 @@
                         android:textSize="14sp" />
 
                     <FrameLayout
-                        android:id="@+id/btn_vbat_connect_account"
+                        android:id="@+id/layout_vbat_connect_account"
```

Comment:
This FrameLayout contains only one button , I hope this ViewGroup is not required